model2/NETBALAN_P80.DATA

```diff
@@ -0,0 +1,360 @@
+-- This reservoir simulation deck is made available under the Open Database
+-- License: http://opendatacommons.org/licenses/odbl/1.0/. Any rights in
+-- individual contents of the database are licensed under the Database Contents
+-- License: http://opendatacommons.org/licenses/dbcl/1.0/
+
+-- Copyright (C) 2022 Equinor
+
+
+-- ------------------------------------------------------------------------
+-- ----------------------- NETWORK TESTING -----------------------------------
+-- ------------------------------------------------------------------------
+
```

Comment:
Could you add a couple of lines here to explain the features we're testing in this model?  Just "network testing" is a little too broad in my opinion.